chacha20_poly1305/src/lib.rs

```diff
@@ -3,7 +3,6 @@
 //! Combine the ChaCha20 stream cipher with the Poly1305 message authentication code
 //! to form an authenticated encryption with additional data (AEAD) algorithm.
 #![no_std]
-
```

Comment:
In ff02845efed31c7319c85a8422734aaabee9126a:

Did rustfmt really do this? I cannot imagine a scenario where this would improve a codebase.

I'll let it go, but c'mon.

Reply:
Yes it merges all `#!` comments together into a single block. Kind of the same as how all exports are merged together into a single block too and why we have all these lines

```
#[rustfmt::skip]                // Keep public re-exports separate.
```

Which is an ugly hack that I am starting to not like because it stops the marked block being formatted so slowly over time the formatting deteriates.

> Yeah, and it would be nice if rustfmt wasn't a piece of shit. But we live in the world we do.

That gave me an enjoyable laugh first thing in the morning.